packages/commonwealth/package.json

```diff
@@ -54,6 +54,7 @@
     "unit-test:watch": "NODE_ENV=test ts-mocha --opts test/mocha-dev.opts './test/unit/**/*.spec.ts' -w --watch-files '**/*.ts'",
     "wait-server": "chmod +x ./scripts/wait-server.sh && ./scripts/wait-server.sh",
     "refresh-all-memberships": "npx ts-node -r tsconfig-paths/register -T ./scripts/refresh-all-memberships.ts",
+    "generate-stakeholder-groups": "npx ts-node -r tsconfig-paths/register -T ./scripts/generate-stakeholder-groups.ts",
```

Comment:
As is currently, the deployment plan will fail on prod. We have two options here:

1. You need to add ts-node and tsconfig-paths to the regular dependencies.
2. Update the deployment plan to: `yarn add ts-node tsconfig-paths && yarn generate-stakeholder-groups`

Also you should double check by testing on staging to be 100% sure this still doesn't fail (Because there may be other dev-dependencies that we need to make regular dependencies).

Reply:
All these scripts are better served by a separate workspace (something like `/packages/scripts`), with its own dependencies... given we create the commands in libs instead of controllers. We should probably start thinking about this for any new features instead of continuing with the legacy couplings.